.github/workflows/busted.yml

```
@@ -0,0 +1,43 @@
+name: Busted
+
+on: [ push, pull_request ]
+
+jobs:
+
+  busted:
+    strategy:
+      fail-fast: false
+      matrix:
+        luaVersion: [ "5.4", "5.3", "5.2", "5.1", "luajit" ] # , "luajit-openresty" ]
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup ‘lua’
+        uses: leafo/gh-actions-lua@v9
+        with:
+          luaVersion: ${{ matrix.luaVersion }}
+
+      - name: Setup ‘luarocks’
+        uses: leafo/gh-actions-luarocks@v4
+
+      - name: Setup dependencies
+        run: |
+          luarocks install busted
+          luarocks install luacov-coveralls
+
+      - name: Replace system say with self
+        run: |
+          luarocks remove say --force
+          luarocks make
+
+      - name: Run regression tests
+        run: busted -c -v
```

```suggestion
        # disable the paths to force use of installed rockspec
        run: busted -c -v --lpath="" --cpath="" --Xoutput "--color"'
```

I [recently started using this](https://github.com/Tieske/project.lua/blob/main/Makefile#L36-L40). This will disable the LUA_PATH to the local source files, and will only test as it is installed. This will ensure there are failures if some new file is not added to the rockspec for example.

Also adds color (with new busted version) in CI.

---

Also, should we add `--coverage` here, and remove it from the `.busted` since it has no use when testing locally, it just slows down.

---

The `-c` here *is* `--coverage`, but yes I'll drop it from `.busted`. I'm actually not really a fan of having that file at all since all the arguments are basically a matter of preference (and they don't happen to match mine). None of them are really project specific or required for function.